src/curl_async.c

```diff
@@ -173,7 +173,7 @@ static void reevaluate_number_of_parallel_downloads(struct swupd_curl_parallel_h
 		h->max_xfer = 1;
 	}
 
-	printf("Reducing number of parallel downloads to %ld\n", h->max_xfer);
+	print("Reducing number of parallel downloads to %ld\n", h->max_xfer);
```

Comment:
looks like most other messages use lower case for the first letter

Reply:
I am unsure about that. Right now it is using lower case when there's a prefix, but I think we should change that. What do you prefer?

    Error: download failed
    Error: Download failed

Reply:
I think I like "Error: Download failed" better

Reply:
I prefer that too. I'll fix that.